src/main/g8/backend/src/main/resources/application.conf

```diff
@@ -0,0 +1,12 @@
+$deviceType;format="norm"$ {
+  # The app will track the latest readings from upto
+  # this many sensors.
+  maximum-nr-of-sensors = 10000
+
+  # The app will update snapshots in storage if they
+  # are this many events behind.
+  save-interval = 1000
```

Comment:
I wonder if we should modify this to 100, which in particular may save load in lower specced devices. I know it's at 1000 elsewhere, so I'm suggesting changing everywhere I suppose.